.github/workflows/pkgTests.yml

```diff
@@ -46,9 +45,9 @@ jobs:
       env:
         sirene_key: ${{ secrets.SIRENE_KEY }}
       run: |
-        pytest -v --cov -n auto
+        pytest -v --cov --cov-report=xml -n auto

```

Comment:
I have not run this command (yet), but from previous experience, this might include coverage on tests themselves. I had to run ` run pytest --cov-config pyproject.toml --cov`, and in the pyproject.toml add the following config:
```
[tool.coverage.run]
omit = [
    "tests/*",
    ]
```

Can you double check on that?